snp_motion_planning/config/task_composer_plugins.yaml

```
@@ -1,4 +1,14 @@
 # Anchors
+
+foo: &raw_inputs
+  program: input_data
+  environment: environment
+  profiles: profiles
+bar: &mod_inputs
+  program: output_data
+  environment: environment
+  profiles: profiles
```

```suggestion
raw: &raw_inputs
  program: input_data
  environment: environment
  profiles: profiles
mod: &mod_inputs
  program: output_data
  environment: environment
  profiles: profiles
```
Let's change these keys to something less arbitrary